### ISSUE TITLE
[Request] Zerotier

### ISSUE THREAD

**Issue by michaelsmoody** [WDCommunity/wdpksrc#39]
Unfortunately, the builds for Zerotier don't work on the EX4, probably due to the CPU difference. Please see: https://github.com/zerotier/ZeroTierNAS/issues/24

If anyone would be willing to create a zerotier release, I would be immensely grateful. I ask simply due to others having experience with this toolchain. If not, please let me know, and I'll take a look to see if I can make it work (but I'll be starting from scratch).

Thanks again,
Michael

**Comment by stefaang** [comment]
Oh the dreaded EX4... it has the most ancient toolchain of all My Cloud products.
It features (eg)libc v2.8 (released in 2009) so there's several other projects refusing to compile as well. 
I'm not inclined to spend time on it, sorry.

People can simply install debian, see the links in the wiki.

**Comment by stefaang** [comment]
ZeroTier says it requires at least gcc 4.8.2 to compile, while the EX4 toolchain ships with the old gcc 4.4.7 ...

    arm-mv5sft-linux-gnueabi.tgz  
    arm-sddnd-linux-gnueabi-gcc-4.4.7+glibc-2.8.tar.gz

**Comment by michaelsmoody** [comment]
It's a shame, really. I guess I'll be looking into a NAS upgrade at some point in the near-ish future. I'll likely be moving to QNAP, as WD simply had little interest in making the MyCloud NAS terribly useful.

**Comment by joseph-henry** [comment]
Hello @stefaang  and @michaelroland. I am the package maintainer for the ZeroTier NAS packages and in an attempt to find a good cross-compiling solution I found this repo (Nice work!). To my surprise, this issue exists!

I've taken a look and the compiler toolchain is indeed too old to support our package without gutting significant portions of our service. Thus, I have added new pre-built packages which can be found here: 

http://download.zerotier.com/dist/wd/

Let me know if anyone experiences any issues or has any feature requests.
